### ISSUE TITLE
Update aria-hidden on parent menu item focus/hover

### ISSUE THREAD

**Issue by jasonnrosa** [HubSpot/cms-theme-boilerplate#224]
**Description**

The boilerplate currently gets flagged for having `[aria-hidden="true"]` elements contain focusable descendants (`ul.submenu.level-2`). 

**Expected behavior**

We should update that attribute on hover/focus via JavaScript to be set to `false`. 

**Checklist**

- [X] I have read the [CONTRIBUTING](https://github.com/HubSpot/cms-theme-boilerplate/blob/master/CONTRIBUTING.md) document.
- [X] I have checked the [issue tracker](https://github.com/HubSpot/cms-theme-boilerplate/issues) to make sure there isn’t already an open issue.

**Comment by TanyaScales** [comment]
Currently the use of `aria-hidden="true"` on the element is misused. We should be using a combination of `aria-haspopup="true" aria-expanded="false"` that changes to `aria-haspopup="true" aria-expanded="true"` when a user is focuses/hovers, etc. on the parent menu element. This will remediate the issue for accessibility and show the true state of the menu to assistive technology users.
